src/Client.ts

```diff
@@ -251,7 +258,7 @@ export default class Client<ContentTypes = any> {
   address: string
   keystore: Keystore
   apiClient: ApiClient
-  contacts: Set<string> // address which we have connected to
+  contacts: Contacts
```

Comment:
We just have to be very sure that this is a drop-in replacement for the functionality we were using the Contacts set for previously. Mostly just for deciding whether to send an intro to someone in a V1 conversation.

Should be fine.

Reply:
yep, it has _very_ limited usage.